carbon-design-system/carbon
# Add TypeScript types to `FilterableMultiSelect`

Closes #12546

#### Changelog

**New**

- Added `useCombobox` from Downshift

**Changed**

- Converted FilterableMultiSelect file from JS to TSX.

**Removed**

- Removed `<Downshift/>` component

#### Testing / Reviewing

- Make sure the functionality remains the same.
- Check if the types make sense and we didn't miss anything.
- CI checks should pass
